classes/tl_form.php

```diff
@@ -89,16 +92,49 @@ public function generateTokens(array $arrData, array $arrForm, array $arrFiles,
             }
         }
 
+        // Add formconfig.
         foreach ($arrForm as $k => $v) {
-            \Haste\Util\StringUtil::flatten($v, 'formconfig_'.$k, $arrTokens, $delimiter);
+            \Haste\Util\StringUtil::flatten($v, 'formconfig_' . $k, $arrTokens, $delimiter);
```

Comment:
```suggestion
            \Haste\Util\StringUtil::flatten($v, 'formconfig_'.$k, $arrTokens, $delimiter);
```
Please do not change any code unrelated to this PR.